SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialRPCService.h

```diff
@@ -50,15 +50,16 @@ enum class EPushRPCResult : uint8
 	QueueOverflowed,
 	DropOverflowed,
 	HasAckAuthority,
-	NoRingBufferAuthority
+	NoRingBufferAuthority,
+	NoEntityInStaticComponentView
 };
 
 class SPATIALGDK_API SpatialRPCService
 {
 public:
 	SpatialRPCService(ExtractRPCDelegate ExtractRPCCallback, const USpatialStaticComponentView* View, USpatialLatencyTracer* SpatialLatencyTracer);
 
-	EPushRPCResult PushRPC(Worker_EntityId EntityId, ERPCType Type, RPCPayload Payload);
+	EPushRPCResult PushRPC(Worker_EntityId EntityId, ERPCType Type, RPCPayload Payload, bool bCreatedEntity = false);
```

Comment:
Why does this have a default value? It seems like this information should always be available to the caller.

Reply:
Updated. Had to also update `RPCServiceTest.cpp`.